src/main/scala/inox/evaluators/Evaluator.scala

```diff
@@ -4,6 +4,7 @@ package inox
 package evaluators
 
 object optIgnoreContracts extends FlagOptionDef("ignore-contracts", false)
+object optCrashOnChoose extends FlagOptionDef("crash-on-choose", false)
```

Comment:
Nit: eval-quantifiers?

Reply:
Better name indeed! Thanks. Should I expose this option in the CLI, or do we keep only for internal usage?

Reply:
As you prefer.